### PR TITLE
Feature/board_class_tests

### PR DESCRIPTION
**Pull request by reneemes**
I wrote out some of the tests for the board class. Seems like this #valid_placement? method is going to be tricky to work though. Let me know what you think about this so far.
Late night coding, so let me know if I am missing anything lol

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -33,6 +33,8 @@ def valid_coordinate?(coordinate)
 
     def valid_placement?(ship_name, desired_coordinates)
        desired_coordinates.length == ship_name.length
+        range = "A1".."A4", "B1".."B4", "C1".."C4", "D1".."D4"
```
Comment (TDManning) [inline lib/board.rb]: This looks good. Its passing the test.

File: lib/board.rb [hunk 2]
```diff
@@ -33,8 +33,8 @@ def valid_coordinate?(coordinate)
 
     def valid_placement?(ship_name, desired_coordinates)
        desired_coordinates.length == ship_name.length
-        range = "A1".."A4", "B1".."B4", "C1".."C4", "D1".."D4"
-        require 'pry'; binding.pry
+        # range = "A1".."A4", "B1".."B4", "C1".."C4", "D1".."D4"
```
Comment (TDManning) [inline lib/board.rb:36]: I see your modified method.

**Comment by TDManning** [comment]
This merge updates the board class with the valid_coordinate? method. Tests have also been updated. Nice work Renee!

File: lib/board.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            "D4" => nil
+        }
+    end
+
+    def valid_coordinate?(coordinate)
+        if @cells.include?(coordinate)
+            true
+        else
+            false
+        end
```
Comment (CPPittman) [inline lib/board.rb:31]: The if else looks like it will properly check the validity of the coordinates

**Pull request merged**